client/signup/steps/import-from/components/error-message/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { BackButton, NextButton, SubTitle, Title } from '@automattic/onboarding';
+import { createElement, createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import page from 'page';
+import React from 'react';
+import { getStepUrl } from 'calypso/signup/utils';
+
+import './style.scss';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+interface Props {
+	siteSlug: string;
+}
+
+const ErrorMessage: React.FunctionComponent< Props > = ( props ) => {
+	const { __ } = useI18n();
+	const { siteSlug } = props;
+
+	/**
+	 ↓ Methods
+	 */
+	const backToStart = (): void => {
+		page( getStepUrl( 'importer', 'capture', '', '', { siteSlug } ) );
+	};
+
+	const backToIntent = (): void => {
+		page( getStepUrl( 'setup-site', 'intent', '', '', { siteSlug } ) );
+	};
+
+	return (
+		<div className="import-layout__center">
+			<div className="import__header">
+				<div className="import__heading import__heading-center">
+					<Title>
+						{ createInterpolateElement( __( 'Oops, <br />something went wrong.' ), {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Oops, something went wrong.' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/onboarding-wix-improvements)